go/exocom/src/client_registry/subscription_manager.go

```diff
@@ -0,0 +1,85 @@
+package clientRegistry
+
+import (
+	"github.com/Originate/exocom/go/exocom/src/message_translator"
+)
+
+// Subscriber is the combination of a client name and internal namesapce
+type Subscriber struct {
+	ClientName        string
+	InternalNamespace string
+}
+
+// SubscriptionManager manages what clients should be sent messages
+type SubscriptionManager struct {
+	MessageNameToSubscribers map[string][]Subscriber
+	Routing                  map[string]Route
+}
+
+// NewSubscriptionManager returns a SubscriptionManager for the given routes
+func NewSubscriptionManager(routes map[string]Route) *SubscriptionManager {
```

Comment:
Can we defined a `Routes` type that is `map[string]Route` to make the semantics of data more explicit and not juggle so many anonymous maps around?